### PR TITLE
Crandb

### PR DESCRIPTION
**Pull request by maelle**
cf #15 

Should I tackle the README update (i.e. adding crandb functions) in this same PR?

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -23,3 +26,18 @@ Suggests:
...
+    'config.r'
+    'couch.r'
+    'cran.R'
+    'crandb-internal-api.r'
+    'crandb-public-api.r'
+    'pkgsearch-package.R'
+    'print.R'
+    'update.r'
+    'utils-pipe.R'
+    'utils.R'
```
Comment (gaborcsardi) [inline DESCRIPTION]: Can we move all files to `.R` instead of `.r`?

File: NAMESPACE [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
 S3method(summary,pkg_search_result)
+S3method(summary,r_releases)
+export("%>%")
+export(events)
+export(list_packages)
 export(more)
+export(package)
 export(pkg_search)
 export(ps)
+export(releases)
```
Comment (gaborcsardi) [inline NAMESPACE]: We'll need to rename these to `cran_events()`, `cran_package()`, etc. or `pkg_events()` or sg. I guess `cran_events()`.

Comment (maelle) [reply NAMESPACE]: I renamed three functions, `cran_releases()`, `cran_events()` and `cran_package()`, is it fine?

**Comment by maelle** [comment]
CI does not approve :wink:

**Pull request merged**